README.md

```diff
@@ -23,14 +23,13 @@ To try out the latest release of the SDK go to our [Cordova Showcase Template Ap
 
 This Aerogear SDK consists of a set of Modules.
 
-- [Auth](./docs/modules/ROOT/pages/auth.adoc): Mobile Application Auth Module
-- [Metrics](./docs/modules/ROOT/pages/metrics.adoc): Mobile Application Metrics Module
-- [Security](./docs/modules/ROOT/pages/security.adoc): Mobile Application Security Module
+- [Auth](./docs/modules/ROOT/pages/using-auth-sdk.adoc): Mobile Application Auth Module
```

Comment:
@ciaranRoche We are not linking to specific pages in the[ Android Readme](https://github.com/aerogear/aerogear-android-sdk#list-of-modules) to avoid links getting out of date and needing updating. There is the github link in the table to the pages folder so this is enough. Let's match the Android syntax  here also.